src/SpectatorView.UniversalSpectator.Unity/Assets/ARKit-Unity-Plugin

```diff
@@ -0,0 +1 @@
+../../.././external/ARKit-Unity-Plugin
```

Comment:
I think you should name this UnityARKitPlugin, as that's the directory name that the ASA script tries to hide / show when changing to the iOS platform

Reply:
Oh, you're right.  Thanks for the catch!  I was a bit confused, because the ASA plugin used one name, but the "add dependencies" script used another.  Perhaps a separate PR should be to make the "add dependencies" script and SpectatorView.Example.Unity use the same name as ASA.